<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="sigverse_ros_bridge_port"        default="50001" />
	<arg name="ros_bridge_port"                 default="9090" />

	<node pkg="competition_test_tools" exec="interactive_cleanup_sample" output="screen" launch-prefix="gnome-terminal --geometry=80x30+100+100 -t 'Interactive Cleanup' -- ">
		<!--
		<remap from="/interactive_cleanup/message/to_robot"   to="/interactive_cleanup/message/to_robot" />
		<remap from="/interactive_cleanup/message/to_human"   to="/interactive_cleanup/message/to_human" />
		<remap from="/hsrb/command_velocity"                  to="/hsrb/command_velocity" />
		<remap from="/hsrb/arm_trajectory_controller/command" to="/hsrb/arm_trajectory_controller/command" />
		<remap from="/hsrb/gripper_controller/command"        to="/hsrb/gripper_controller/command" />
		-->
	</node>

	<node pkg="sigverse_ros_bridge" exec="sigverse_ros_bridge" args="$(var sigverse_ros_bridge_port)" />

	<include file="$(find-pkg-share rosbridge_server)/launch/rosbridge_websocket_launch.xml" > 
		<arg name="port" value="$(var ros_bridge_port)"/>
		<arg name="default_call_service_timeout" value="5.0"/>
		<arg name="call_services_in_new_thread" value="true"/>
		<arg name="send_action_goals_in_new_thread" value="true"/>
<!--		<arg name="bson_only_mode" value="true"/> -->
	</include>
</launch>

